### ISSUE TITLE
Direct user to set up project directory if specified projects_dir doesn't exist

### ISSUE THREAD

**Issue by hbillings** [NUKnightLab/tarbell#15]
Currently throws an OSError.

**Comment by JoeGermuska** [comment]
I don't think we should direct anyone to the command line unless we absolutely can't do that for them from the admin app.

**Comment by hbillings** [comment]
Agreed; currently you can't run the admin app if you have a projects directory set up and it doesn't exist. The admin app needs to be more flexible in how and when you can edit your settings.

**Comment by jywsn** [comment]
I'm not clear on what you want to happen.  Do you want to automatically create the directory?  Show an error and let them fix it themselves?  Take them back through the configuration modal sequence?  Something else?

**Comment by hbillings** [comment]
Closing as part of Heather's Adventures in the Wrong Branch.

**Comment by jywsn** [comment]
This is actually still an issue.  reopening...
